frontend/webpack.config.js

```diff
@@ -33,7 +33,7 @@ const BUILD_FILE_NAMES = {
 
 const PATH_ALIASES = {
   "@": PATHS.src,
-  // Put your aliases here
+  "react-dom": "@hot-loader/react-dom",
```

Comment:
This is weird. It's definitely needed? What does it do?

Reply:
Agreed, this is weird. This was to address this [react-hot-loader issue](https://github.com/gaearon/react-hot-loader/issues/1227) which was showing up in my console. I didn't think the PATH_ALIASES was the best fit, but I wasn't keen on creating a NON_PATH_ALIASES.

Reply:
That link was really helpful, thanks.

Since this patch is a bit different than our regular path aliases, you could do something like this on line 72:
```javascript
resolve: {
      alias: {
         "react-dom": "@hot-loader/react-dom", // patch react-dom import
         ...PATH_ALIASES
      },
    },
```

That just keeps it out of our other aliases. But they do the same thing in the end, so not a big deal to me either way.

Reply:
Sure, that makes sense - I'll move it out of the way.